lib/cc/engine/analyzers/analyzer_base.rb

```diff
@@ -11,6 +11,10 @@ class Base
           ::RubyParser::SyntaxError,
         ]
 
+        DEFAULT_MASS_THRESHOLD = 28
+        BASE_POINTS = 1_500_000
+        POINTS_PER = 50_000
```

Comment:
What do you think of `BASE_POINTS` and `MASS_OVERAGE_POINTS` instead of `POINTS_PER`?

I didn't find the current name intuitive.